### PR TITLE
issue 2956

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2956, added info about Education editions of Windows 10, source: [Windows 10 editions for education customers](https://docs.microsoft.com/education/windows/windows-editions-for-education-customers)

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-application-guard/reqs-wd-app-guard.md [hunk 1]
```diff
@@ -36,6 +36,6 @@ Your environment needs the following software to run Windows Defender Applicatio
 
 |Software|Description|
 |--------|-----------|
-|Operating system|Windows 10 Enterprise edition (including Windows 10 Education), version 1709 or higher<br>Windows 10 Professional edition (including Windows 10 Pro Education), version 1803|
+|Operating system|Windows 10 Enterprise edition, version 1709 or higher<br>Windows 10 Professional edition, version 1803 or higher<br>Windows 10 Education edition, version 1709 or higher<br>Windows 10 Pro Education edition, version 1803 or higher|
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-guard/reqs-wd-app-guard.md]: Copy edited - no changes.

**Comment by mypil** [comment]
@Justinha - can you please assist to check, approve and merge the content for this PR based on issue #2956?

Thank you.

**Comment by mypil** [comment]
@Justinha - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #2956?

Thank you.

**Pull request merged**